source/includes/stackpath/_network_policy_rules.md

```diff
@@ -3,6 +3,114 @@
 Network policy rules allows you to control inbound and outbound traffic to your [workload](#stackpath-workloads).
 
 
+<!-------------------- LIST NETWORK POLICY RULES -------------------->
+### List network policy rules
+
+```shell
+curl -X GET \
+  -H "MC-Api-Key: your_api_key" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules"
```

Comment:
maybe add the workloadId parameter option information to filter for a particular workloadId

Reply:
Fixed here, shouldn't conflict with the edit rule PR https://github.com/cloudops/cloudmc-api-docs/pull/272/commits/91f423d22f68d3e4a32d05b96764d13e97ee1eeb